pydantic_factories/factory.py

```diff
@@ -485,14 +484,24 @@ def build(cls, factory_use_construct: bool = False, **kwargs: Any) -> T:
 
         Note - this is supported only for pydantic models
         """
+        from pydantic_factories.fields import PostGenerated
+
+        generate_post = {}
```

Comment:
this should be typed, namely this is a `Dict[str, PostGenerated]`

Also, please run `pre-commit run --all-files` to ensure you are not seeing mypy issues.